doc/man3/SSL_CTX_set_tls_cookie_generate_cb.pod

```diff
@@ -0,0 +1,60 @@
+=pod
+
+=head1 NAME
+
+SSL_CTX_set_tls_cookie_generate_cb,
+SSL_CTX_set_tls_cookie_verify_cb
+- Callback functions for stateless TLS1.3 cookies
+
+=head1 SYNOPSIS
+
+ #include <openssl/ssl.h>
+
+ void SSL_CTX_set_tls_cookie_generate_cb(SSL_CTX *ctx,
+                                         int (*gen_tls_cookie_cb) (SSL *ssl,
+                                                                   unsigned char
+                                                                   *cookie,
+                                                                   unsigned int
+                                                                   *cookie_len));
+ void SSL_CTX_set_tls_cookie_verify_cb(SSL_CTX *ctx,
+                                       int (*verify_tls_cookie_cb) (SSL *ssl,
+                                                                    const unsigned char
+                                                                    *cookie,
+                                                                    unsigned int
+                                                                    cookie_len));
+
+=head1 DESCRIPTION
+
+SSL_CTX_set_cookie_generate_cb() sets the callback used by L<SSL_stateless(3)>
+to generate the application-controlled portion of the cookie provided to clients
+in the HelloRetryRequest transmitted as a response to a ClientHello with a
+missing or invalid cookie. gen_tls_cookie_cb() must write at most
+SSL_COOKIE_LENGTH bytes into B<cookie>, and must write the number of bytes
+written to B<cookie_len>. If a cookie cannot be generated, a zero return value
+can be used to abort the handshake.
+
+SSL_CTX_set_cookie_verify_cb() sets the callback used by L<SSL_stateless(3)> to
+determine whether the application-controlled portion of a ClientHello cookie is
+valid. A nonzero return value from app_verify_cookie_cb() communicates that the
+cookie is valid. The integrity of the entire cookie, including the
+application-controlled portion, is automatically verified using a HMAC before
```

Comment:
Typo: probably should be "using HMAC"

Reply:
I'm not sure about that. "using hash-based message authentication code" strikes me as incorrect because "code" is singular; it seems to me that it should be either "using a HMAC" or "using HMACs," but the latter would be incorrect because we're discussing a single abstract cookie here.

Reply:
Well I would argue that you are using HMAC here to refer to the name of an algorithm, not a specific (singular) code that that algorithm generates. But I don't care enough about this to hold up the review, so I'll approve this PR.

Reply:
Ah, I see where you're coming from. Tweaked the wording a little.